agent/app/agent_capability_unix_test.go

```diff
@@ -925,3 +925,37 @@ func TestAppendFSxWindowsFileServerCapabilities(t *testing.T) {
 	assert.Equal(t, len(inputCapabilities), len(capabilities))
 	assert.EqualValues(t, capabilities, inputCapabilities)
 }
+
+func TestIsGuardDutySupported(t *testing.T) {
+	testCases := []struct {
+		name                         string
+		expectedIsGuardDutySupported bool
```

Comment:
both test cases have this set to false, is that the intention?

Reply:
This need to be true, this might have got update during my previous commit. I missed this somehow. updated this